sql/updates/world/3.3.5/2016_08_09_00_world.sql

```diff
@@ -0,0 +1,7 @@
+-- NPC 35364 Slahtz <Experience Eliminator> / 35365 Behsten <Experience Eliminator>
+
+UPDATE `creature_template` SET `gossip_menu_id`= 10638 WHERE `entry`= 35365; -- 35364 already has this.
+
+DELETE FROM `gossip_menu_option` WHERE `menu_id`= 10638 AND `id`= 1 AND `OptionBroadcastTextID`= 35532;
```

Comment:
Found DB content in TDB 6.04 `gossip_menu_option` looking better than the 3.3.5 version:
```sql
SELECT * FROM `gossip_menu_option` WHERE `menu_id` IN (10638,10639);
```
```
menu_id      id  option_icon  option_text                                OptionBroadcastTextID  option_id  npc_option_npcflag  action_menu_id  action_poi_id  box_coded  box_money  box_text                                                     BoxBroadcastTextID
-------  ------  -----------  -----------------------------------------  ---------------------  ---------  ------------------  --------------  -------------  ---------  ---------  -----------------------------------------------------------  ------------------
  10638       0           16  I no longer wish to gain experience.                       35531          1                   1               0              0          0     100000  Are you certain you wish to stop gaining experience?                          0
  10638       1           17  I wish to start gaining experience again.                      0          0                   0               0              0          0     100000  Are you certain you wish to start gaining experience again?                   0
```
Scroll to the end of this DB output and see what is missing in the TDB 335.61 version of this gossip.

Time to get this PR fixed now. I cannot use this version of the script because of `OnGossipHello()` conditions.